Triphouse-project/style.css

```diff
@@ -197,13 +197,10 @@ form input {
 
```

Comment:
![2021-04-22_22h03_38](https://user-images.githubusercontent.com/19935417/115771506-b15d7700-a3b6-11eb-96c8-be2e6e21fb7d.png)
Add breakpoint for mobile on that value.

Reply:
Add new breakpoint (859px) for mobile and width 100%.